data-collection/deploy/module-organization.yaml

```diff
@@ -208,19 +208,21 @@ Resources:
                   return path[::-1]
 
               @lru_cache(maxsize=10000)
-              def get_tags(self, id_):
+              def get_tags(self, id_, athena_friendly=False):
                   """returns a dict of tags"""
+                  def _process_key(key, athena_friendly):
+                      return key.replace(':', '_').replace('-', '_').replace(' ', '_').replace('.', '_') if athena_friendly else key
```

Comment:
Should we consider a regex and have we tested all the allowable characters in tags in Athena?
 . : + = @ _ / -

Reply:
https://xkcd.com/1171/

Reply:
re.sub('[^a-zA-Z0-9]', '_', key)
Draconian but sure to be safe

Reply:
added regexp